recipes/prometheus.rb

```diff
@@ -78,6 +78,7 @@
 certificate = "#{crypto_dir}/#{x509_helper.get_certificate_bundle_name(node['hopsmonitor']['user'])}"
 key = "#{crypto_dir}/#{x509_helper.get_private_key_pkcs8_name(node['hopsmonitor']['user'])}"
 hops_ca = "#{crypto_dir}/#{x509_helper.get_hops_ca_bundle_name()}"
+managed_cloud = (node['install']['enterprise']['install'].casecmp? "true" and exists_local("cloud", "default"))
```

Comment:
I don't understand right away what this is doing. Maybe add a comment about the condition?

Reply:
I can add a comment. It's just checking if we are installing the cloud recipe on this node.